phone_number_spec.rb

```diff
@@ -0,0 +1,41 @@
+require_relative "./phone_number"
+
+RSpec.describe PhoneNumber do
+  let(:phone_number) { PhoneNumber.new("5558675309") }
+
+  describe "#number" do
+    it "is a formatted number" do
+      expect(phone_number.number).to eq("(555) 867-5309")
+    end
+  end
+  
+  describe "#to_i" do
+    it "is the number as an int" do
+      new_number = PhoneNumber.new("555-867-5309")
+      expect(new_number.to_i).to eq(5558675309)
+    end
+  end
+
+  describe "#text" do
+    it "calls PhoneActions#text!" do
+      expect_any_instance_of(PhoneActions).to receive(:text!)
```

Comment:
In addition to `#receive` please use [#with](https://rubydoc.info/gems/rspec-mocks/frames#:~:text=the%20original%20methods.-,Expecting%20Arguments,-expect) to validate that the method is actually being invoked with the correct arguments.

This comment also applies for the rest of the tests.

```suggestion
      expect_any_instance_of(PhoneActions).to receive(:text!).with("(555) 867-5309", "hello world!")
```